chore: Add hooli partner demo page (#99)
UnumID/UnumID.github.io: docs/hooli-partner-demo.mdx

---
id: hooli-partner-demo
title: Hooli Partner Demo
sidebar_label: Hooli Partner Demo
slug: /hooli-partner-demo
---

import Tip from '../src/components/Tip';

> The open source Hooli partner demo demonstrates an example implementation of the issuing partner Verified Inc. digital identity cards which then redirects to Kredita. It is an application for a fictional customer, Hooli, which uses Verified Inc. to issue reusable <Tip type="credential">credentials</Tip> to its users and send them to the partner url.

Reviewer: ```suggestion
The open source Hooli partner demo demonstrates an example implementation of issuing Verified Inc. digital identity cards to share with a specific partner. It is an application for a fictional customer, Hooli, which uses Verified Inc. to issue reusable <Tip type="credential">credentials</Tip> to its users and share them with another fictional customer, Kredita.
```

> Additional information on how to issue credentials with a partner can be found in the Guide's issuance with partner [documentation](/issuance-guide#partner-issuance).

Reviewer: ```suggestion
Additional information on how to issue credentials with a partner can be found in the Partner Issuance [guide](/issuance-guide#partner-issuance).```

Author: This was resolved in https://github.com/UnumID/UnumID.github.io/pull/102

## Web Demo
The live Hooli web application demo can be found [here](https://hooli-partner-web.demo.sandbox-verifiedinc.com/login). Its open source code can be found on [Github](https://github.com/UnumID/Hooli-Demo-Web).

> Feel free to use a real email address that you control or our test account, richard@piedpiper.net. For the purposes of the demo, the password field is irrelevant. If opting to use the test account you can use the static OTP of `111111`. For the appropriate experience of demo, it is recommended to use the same e-mail from [Hooli](/holy-demo) and [Kredita](/kredita-demo).

Reviewer: Note: I haven't used the new demo yet, so I'm not sure if we still need the part about using the same email or not.

```suggestion
Feel free to use a real email address that you control, or our test account, richard@piedpiper.net. You can enter any value in the password field. If you used a real email address, we will send you a one-time password to authenticate to the [wallet](https://wallet.sandbox-verifiedinc.com). If you used the test account, your OTP is `111111`.
```

Author: This was resolved in https://github.com/UnumID/UnumID.github.io/pull/102
